uwsgi.h

```diff
@@ -1515,8 +1522,9 @@ struct wsgi_request {
 	// when set, do not send warnings about bad behaviours
 	int post_warning;
 
-	size_t range_from;
-	size_t range_to;
+	enum uwsgi_range range_parsed;
+	int64_t range_from;
```

Comment:
the code is compiled with '-D_LARGEFILE_SOURCE', '-D_FILE_OFFSET_BITS=64' so size_t should be always 64bit